Ical.Net/Evaluation/RecurrencePatternEvaluator.cs

```diff
@@ -192,10 +192,11 @@ private IEnumerable<CalDateTime> EnumerateDates(CalDateTime originalDate, CalDat
                 dateCount++;
             }
 
-            noCandidateIncrementCount++;
             if (noCandidateIncrementCount > options?.MaxUnmatchedIncrementsLimit)
                 throw new EvaluationLimitExceededException();
 
+            noCandidateIncrementCount++;
```

Comment:
oh yes, sure

Reply:
Its not critical at all, as the property really is intended to be used in a heuristical manner. That it was wrong only got obvious when setting the value to 0, which should work fine, as long as no iterations are skipped, but didn't. Thanks for the review!